src/test/java/persistence/sql/dml/query/DeleteByIdQueryBuilderTest.java

```diff
@@ -0,0 +1,17 @@
+package persistence.sql.dml.query;
+
+import org.junit.jupiter.api.Test;
+import persistence.sql.Person;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DeleteByIdQueryBuilderTest {
```

Comment:
```suggestion
class DeleteByIdQueryBuilderTest {
```
junit 5부터는 테스트 클래스에 pulibc을 제거해주셔도 됩니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods